### DISCUSSION TITLE
06AA2998-6420-5339-71F7-C808E9C1514E

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#177] [General]
# 06AA2998-6420-5339-71F7-C808E9C1514E

Asterix is the first French satellite. It was launched on 26 November 1965 by a Diamant A rocket from the CIEES launch site at Hammaguir, Algeria. With Asterix France became the sixth country to have an artificial satellite and the third country to launch a satellite on its own rocket. Its main purpose was to test the Diamant launcher, though it was also designed to study the ionosphere. Asterix orbits Earth as of 2022 and is expected to do so for centuries. But communication with the satellite was lost just 2 days after its launch.

https://celestia.mobi/addon?item=06AA2998-6420-5339-71F7-C808E9C1514E

**Comment by OlivaxNoel** [comment]
Frances first ever satellite
